cmd/common.go

```diff
@@ -147,8 +77,11 @@ func filterLatestProfiles(profiles []profileutil.ProvisioningProfileInfoModel) [
 	return filteredProfiles
 }
 
-func collectIpaExportCodeSignGroups(tool Tool, archive xcarchive.IosArchive, installedCertificates []certificateutil.CertificateInfoModel, installedProfiles []profileutil.ProvisioningProfileInfoModel) ([]export.IosCodeSignGroup, error) {
-	iosCodeSignGroups := []export.IosCodeSignGroup{}
+func collectIpaExportCodeSignGroups(tool Tool, archive Archive, installedCertificates []certificateutil.CertificateInfoModel, installedProfiles []profileutil.ProvisioningProfileInfoModel) ([]interface{}, error) {
+	iosCodeSignGroups := []interface{}{}
```

Comment:
this is not the best.
i think we should introduce a new interface for the codesigngroups.

I would modify the [ios](https://github.com/bitrise-tools/go-xcode/blob/master/export/ios.go#L12) and [macos](https://github.com/bitrise-tools/go-xcode/blob/master/export/mac.go#L10) codesigngroups.

Both have the same properties:

```
	Certificate        certificateutil.CertificateInfoModel
	BundleIDProfileMap map[string]profileutil.ProvisioningProfileInfoModel
```

make them unexported and create a function to access them.

macos has an extra field:

```
InstallerCertificate *certificateutil.CertificateInfoModel
```

i would recommend to also make this unexported and create an accessor function for this and also create it for ios and always return nil. This way we can define a common interface for them:

```
Codesigngroup {
	Certificate() certificateutil.CertificateInfoModel
	BundleIDProfileMap() map[string]profileutil.ProvisioningProfileInfoModel
	InstallerCertificate() *certificateutil.CertificateInfoModel
}
```